docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-metrics-ui.asciidoc

```diff
@@ -0,0 +1,68 @@
+[role="xpack"]
+[[ootb-ml-jobs-metrics-ui]]
+= Metrics {anomaly-detect} configurations
+++++
+<titleabbrev>Metrics</titleabbrev>
+++++
+
+// tag::metrics-jobs[]
+These {anomaly-jobs} can be created in the
+{kibana-ref}/xpack-infra.html[Metrics app] in {kib}. When you create one of 
+these jobs, you need to select a field that splits the data to establish 
```

Comment:
> When you create one of these jobs...

IMO this should go into the Observability Guide's page for these jobs (equivalent to the one for Logs app here: https://www.elastic.co/guide/en/observability/master/inspect-log-anomalies.html#enable-anomaly-detection)

Reply:
@szabosteve @lcawl I agree. Created an [issue](https://github.com/elastic/observability-docs/issues/136) for  this in the Observability docs repo.

Reply:
I removed the configuration info as `Inspect metrics anomalies` will contain it.